src/style.css

```diff
@@ -1,3 +1,85 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+
+  --main-color: #00acdc;
+}
+
+li {
+  list-style: none;
+  color: #000;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
```

Comment:
consider using variables for repeating colors

Reply:
Updated via v1.1 commit, please check once more.

Thanks!